wormhole-connect/src/config/types.ts

```diff
@@ -88,6 +88,8 @@ export type IsRouteSupportedHandler = (
   transferDetails: TransferDetails,
 ) => Promise<boolean>;
 
+export type IsTokenSupportedHandler = (token: Token) => boolean;
```

Comment:
Do we want to keep `Token` as an internal type and instead pass `TokenId` or something else here?

Reply:
It might be useful to pass `Token` though since it has `tokenBridgeOriginalTokenId` for example and `symbol`, etc.